homeassistant/components/habitica/util.py

```diff
@@ -59,3 +63,16 @@ def entity_used_in(hass: HomeAssistant, entity_id: str) -> list[str]:
     used_in = automations_with_entity(hass, entity_id)
     used_in += scripts_with_entity(hass, entity_id)
     return used_in
+
+
+def get_config_entry(hass: HomeAssistant, entry_id: str) -> ConfigEntry:
+    """Return config entry or raise if not found or not loaded."""
+    entry: ConfigEntry | None
+    if not (
+        entry := hass.config_entries.async_get_entry(entry_id)
+    ) or entry not in hass.config_entries.async_loaded_entries(DOMAIN):
```

Comment:
We should raise a different error if the entry is not loaded, otherwise it looks really weird in the logs when you call a real config entry which did not load